pkg/image/oci/registry_provider.go

```diff
@@ -98,6 +91,10 @@ func prepareRemoteOptions(ref name.Reference, registryOptions *image.RegistryOpt
 	authenticator := registryOptions.Authenticator(ref.Context().RegistryStr())
 	if authenticator != nil {
 		opts = append(opts, remote.WithAuth(authenticator))
+	} else {
+		// use the Keychain specified from a docker config file.
+		log.Debugf("no registry credentials configured, using the default keychain")
```

Comment:
maybe `Infof`?

Reply:
my vote would be to keep it as debug. (my mental model: info = let me know big-picture execution stages, debug = tell me more about decision points)